meetings/2018-08-14.md

```diff
@@ -0,0 +1,65 @@
+Decorators call agenda and notes
+Tuesday, August 14, 2018, 17:00 UTC
+
+Attendees:
+* Justin Ridgewell (JRL) 
+* Jordan Harband (JH) 
+* Ron Buckton (RB) 
+* Ryan Cavanaugh (RC) 
+* Diego Ferreiro Val (DFV) 
+* Igor Minar (IMR) 
+* Brad Green (BG) 
+
+Agenda & notes:
+* Decorators in toString? 
+    * RB: If you toString a static method, it doesn’t include static. I liken this to how export is not included. It’s just the grammar production that determines the toString. What does including decorators in the toString provide us? One is diagnostic functions, the other is re-evaling can sometimes work. 
+    * RB: I’m a bit concerned with including decorators in toString. There is some value for daignostics, but it has same pros/cons in that it’s limited only in identifying the function. I don’t feel that either is very strong. 
+    * DFV: We’re using toString for lazy evaluation which saves a few 100ms 
+    * RB: When I say it’s brittle, I mean that the class result of the declaration is the same class as what you’ve decoratored. You can replace it. 
+    * RB: A lot of cases we’ve done actually replace the constructor. 
+    * DFV: Not including decorator will break reifing later. 
+    * DH: Same lexical environment won’t always allow you to re-eval. The decorator could be non-pure, etc. 
+    * RB: If we do include, why are we not including export? 
+    * JH: It modiefies the module, not the value. You can’t introspect the value to determine if it’s exported. 
```

Comment:
modiefies -> modifies?

Reply:
@kt3k: would you submit a pull request to fix the mispellings?

Reply:
Yes! #142